gemfirexd/client/src/main/java/io/snappydata/jdbc/TomcatConnectionPool.java

```diff
@@ -0,0 +1,168 @@
+        datasource = new DataSource();
+        datasource.setPoolProperties(poolProperties);
+    }
+
+    /**
+     * Method responsible for collecting pooled properties from the
+     * properties object passed to connection and creates PoolProperties
+     * object by setting the pool properties into it.
+     *
+     * @param prop
+     * @return
+     */
+    private PoolProperties setPoolProperties(Properties prop) {
+
+        PoolProperties poolProperties = new PoolProperties();
+        String url = prop.getProperty(PoolProps.URL.key);
+        poolProperties.setUrl(url);
+        String driverClassName = prop.getProperty(PoolProps.DRIVER_NAME.key);
+        poolProperties.setDriverClassName(driverClassName);
+
+        String username = prop.getProperty(PoolProps.USER.key);
+        if (!Util.isEmptyString(username)) {
+            poolProperties.setUsername(username);
+        }
+
+        String password = prop.getProperty(PoolProps.PASSWORD.key);
+        if (!Util.isEmptyString(password)) {
+            poolProperties.setPassword(password);
+        }
+
+        String initSize = prop.getProperty(PoolProps.INIT_SIZE.key, PoolProps.INIT_SIZE.defValue);
+        poolProperties.setInitialSize(Integer.parseInt(initSize));
+
+        String maxActive = prop.getProperty(PoolProps.MAX_ACTIVE.key, PoolProps.MAX_ACTIVE.defValue);
+        poolProperties.setMaxActive(Integer.parseInt(maxActive));
+
+        String maxIdle = prop.getProperty(PoolProps.MAX_IDLE.key, PoolProps.MAX_IDLE.defValue);
+        poolProperties.setMaxIdle(Integer.parseInt(maxIdle));
+
+        String minIdle = prop.getProperty(PoolProps.MIN_IDLE.key, PoolProps.MIN_IDLE.defValue);
+        poolProperties.setMaxIdle(Integer.parseInt(minIdle));
+
+        String waitTime = prop.getProperty(PoolProps.MAX_WAIT.key,
+                PoolProps.MAX_WAIT.defValue);
+        poolProperties.setMaxWait(Integer.parseInt(waitTime));
+
+        String removeAbandoned = prop.getProperty(PoolProps.REMOVE_ABANDONED.key,
+                PoolProps.REMOVE_ABANDONED.defValue);
+        poolProperties.setRemoveAbandoned(Boolean.parseBoolean(removeAbandoned));
+
+        String removeAbandonedTimeout = prop.getProperty(PoolProps.REMOVE_ABANDONED_TIMEOUT.key,
+                PoolProps.REMOVE_ABANDONED_TIMEOUT.defValue);
+        poolProperties.setRemoveAbandonedTimeout(Integer.parseInt(removeAbandonedTimeout));
+
+        String evictionRunMillis = prop.getProperty(PoolProps.TIME_BETWEEN_EVICTION_RUNS_MILLIS.key,
+                PoolProps.TIME_BETWEEN_EVICTION_RUNS_MILLIS.defValue);
+        poolProperties.setTimeBetweenEvictionRunsMillis(Integer.parseInt(evictionRunMillis));
+
+        String minEvictableIdleTimeMillis = prop.getProperty(PoolProps.MIN_EVICTABLE_IDLE_TIME_MILLIS.key,
+                PoolProps.MIN_EVICTABLE_IDLE_TIME_MILLIS.defValue);
+        poolProperties.setMinEvictableIdleTimeMillis(Integer.parseInt(minEvictableIdleTimeMillis));
```

Comment:
break the long lines above (can see 2-3 which exceed 100 chars)

Reply:
Corrected